test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+describe('Login Page', () => {
+    it('should return 200 for valid credentials', async () => {
+        const res = await fetch('http://localhost:3000/api/login', {
+            method: 'POST',
```

Comment:
The test cases are making actual calls to the API. Consider mocking the API responses to make the tests more isolated and reliable.